services/cii-best-practices/cii-best-practices.service.js

```diff
@@ -0,0 +1,162 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const {
+  colorScale,
+  coveragePercentage: coveragePercentageColor,
+} = require('../../lib/color-formatters')
+
+const ciiBestPracticesSchema = Joi.object({
+  badge_level: Joi.string().required(),
+  tiered_percentage: Joi.number().required(),
+}).required()
+
+const keywords = ['cii', 'cii best practices', 'core infrastructure initiative']
+
+module.exports = class CIIBestPracticesService extends BaseJsonService {
+  static render({ label, message, color }) {
+    return {
+      label,
+      message,
+      color,
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { color: 'blue', label: 'cii' }
+  }
+
+  static get category() {
+    return 'other'
```

Comment:
I had no clue on this one so definitely defer to whatever you all think is best. I spent _just_ enough time reviewing CII to be able to write the badge service for it but I still know next to nothing about it :smile: 

It does seem like CII is an amalgamation of things similar to other badges under the `build` category so that seems logical to me FWIW

Reply:
I switched the category to `build`